### PR TITLE
asset: add export to BigQuery

### PR DESCRIPTION
**Pull request by lvvvvvf**

### CODE REVIEW

**Comment by lvvvvvf** [comment]
> Will you fix the merge conflict? Pull in the latest changes from master, resolve conflicts, then run `go mod tidy`.
> 
> It seems a little odd to show exporting to both in the same sample. Should this be a separate sample? Are we making the same change for all languages?

Done,  Thanks

File: asset/quickstart/export-assets-bigquery/main.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	op, err := client.ExportAssets(ctx, req)
+	if err != nil {
+		log.Fatal(err)
+	}
+	resp, err := op.Wait(ctx)
+	if err != nil {
+		log.Fatal(err)
+	}
+	fmt.Print(response_bq)
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main.go]:
```suggestion
	fmt.Print(resp)
```

File: asset/quickstart/export-assets-bigquery/main_test.go [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	tc := testutil.SystemTest(t)
+	os.Setenv("GOOGLE_CLOUD_PROJECT", tc.ProjectID)
+
+	ctx := context.Background()
+	// Creates a bigquery client.
+	client_bq, err_bq := bigquery.NewClient(ctx, tc.ProjectID)
+	if err_bq != nil {
+		t.Fatalf("failed to create bigquery client: %v", err_bq)
+	}
+	datasetID := strings.ReplaceAll(fmt.Sprintf("%s-for-assets", tc.ProjectID), "-", "_")
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main_test.go]:
```suggestion
	datasetID := strings.Replace(fmt.Sprintf("%s-for-assets", tc.ProjectID), "-", "_", -1)
```

File: asset/quickstart/export-assets-bigquery/main.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+)
+
+func main() {
+	ctx := context.Background()
+	projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+	client, err := asset.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("asset.NewClient: %v", err)
+	}
+	datasetID := strings.ReplaceAll(fmt.Sprintf("%s-for-assets", projectID), "-", "_")
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main.go]:
```suggestion
	datasetID := strings.Replace(fmt.Sprintf("%s-for-assets", projectID), "-", "_", -1)
```

File: asset/quickstart/export-assets-bigquery/main.go [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	datasetID := strings.Replace(fmt.Sprintf("%s-for-assets", projectID), "-", "_", -1)
+	dataset := fmt.Sprintf("projects/%s/datasets/%s", projectID, datasetID)
+	table := "test"
+	req := &assetpb.ExportAssetsRequest{
+		Parent: fmt.Sprintf("projects/%s", projectID),
+		OutputConfig: &assetpb.OutputConfig{
+			Destination: &assetpb.OutputConfig_BigqueryDestination{
+				BigqueryDestination: &assetpb.BigQueryDestination{
+					Dataset: dataset,
+					Table:   string(table),
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main.go]: No need to wrap with `string(...)`. Could even inline the name (`"test"`).

File: asset/quickstart/export-assets-bigquery/main.go [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+				},
+			},
+		},
+	}
+	op, err := client.ExportAssets(ctx, req)
+	if err != nil {
+		log.Fatal(err)
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main.go]:
```suggestion
		log.Fatalf("ExportAssets: %v", err)
```

File: asset/quickstart/export-assets-bigquery/main.go [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	resp, err := op.Wait(ctx)
+	if err != nil {
+		log.Fatal(err)
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main.go]:
```suggestion
		log.Fatal("Wait: %v", err)
```

File: asset/quickstart/export-assets-bigquery/main.go [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	fmt.Print(resp)
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main.go]: Is there anything interesting we can print about the result?

File: asset/quickstart/export-assets-bigquery/main_test.go [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestMain(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	os.Setenv("GOOGLE_CLOUD_PROJECT", tc.ProjectID)
+
+	ctx := context.Background()
+	// Creates a bigquery client.
+	client_bq, err_bq := bigquery.NewClient(ctx, tc.ProjectID)
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main_test.go]:
```suggestion
	client, err := bigquery.NewClient(ctx, tc.ProjectID)
```

File: asset/quickstart/export-assets-bigquery/main.go (hunk 7 above)
Comment (lvvvvvf) [reply asset/quickstart/export-assets-bigquery/main.go]: The test will catch the print and compare with expect

File: asset/quickstart/export-assets-bigquery/main_test.go [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+)
+
+func TestMain(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	os.Setenv("GOOGLE_CLOUD_PROJECT", tc.ProjectID)
+
+	ctx := context.Background()
+	// Creates a bigquery client.
+	client, err := bigquery.NewClient(ctx, tc.ProjectID)
+	if err_bq != nil {
```
Comment (tbpg) [inline asset/quickstart/export-assets-bigquery/main_test.go]:
```suggestion
	if err != nil {
```

(Please make sure the test passes locally.)

Comment (lvvvvvf) [reply asset/quickstart/export-assets-bigquery/main_test.go]: go test -v ./..
go: downloading github.com/bmatcuk/doublestar v1.2.2
=== RUN   TestBadFiles
--- PASS: TestBadFiles (0.02s)
=== RUN   TestLicense
--- PASS: TestLicense (0.27s)
=== RUN   TestRegionTags
--- PASS: TestRegionTags (29.04s)
PASS
ok  	github.com/GoogleCloudPlatform/golang-samples	29.354s

Comment (tbpg) [reply asset/quickstart/export-assets-bigquery/main_test.go]: Looks like there are still variables with underscores in the name. https://golang.org/doc/effective_go.html#mixed-caps

Also, you need to change to this directory before running that command:

```
cd asset/quickstart/export-assets-bigquery
go test -v
```

Comment (lvvvvvf) [reply asset/quickstart/export-assets-bigquery/main_test.go]: Thanks:
go test -v
=== RUN   TestMain
--- PASS: TestMain (65.18s)
PASS
ok  	github.com/GoogleCloudPlatform/golang-samples/asset/quickstart/export-assets-bigquery	65.275s

**Pull request merged**